twat.py

```diff
@@ -315,10 +344,14 @@ def extract_twat(soup, twats, timestamp):
 
 			tweet_id = div.attrs["data-tweet-id"]
 			tweet_user = div.attrs["data-screen-name"]
+			# fetch profile picture
+			fetch_profile_picture(tweet_user, proxies, twhttp=twhttp)
```

Comment:
why is this in extract_twat ? extract_twat is the func thats called to extract _a single tweet_ from the html
i.e. its called 20x per page